ports/rp2/mphalport.h

```diff
@@ -182,11 +182,11 @@ static inline void mp_hal_pin_od_high(mp_hal_pin_obj_t pin) {
 }
 
 static inline void mp_hal_pin_low(mp_hal_pin_obj_t pin) {
-    gpio_clr_mask(1 << pin);
+    gpio_clr_mask64((uint64_t) 1 << pin);
```

Comment:
Might be cleaner to use `1ULL` here instead of casting?  Same for the change below.

Reply:
Actually, even better, use `UINT64_C(1)`.